packages/commercetools/theme/components/MyAccount/BillingAddressForm.vue

```diff
@@ -249,10 +259,27 @@ export default {
       });
     };
 
+    const statesInSelectedCountry = computed(() => {
+      if (form.country) {
+        const selectedCountry = config.countries.find(country => country.name === form.country);
+        if (selectedCountry && selectedCountry.states) {
+          return selectedCountry.states;
+        }
+      }
```

Comment:
Same as above - try to avoid nested `if`s:

```javascript
if (!form.country) {
  return null;
}

const selectedCountry = config.countries.find(({ name }) => name === form.country);

return selectedCountry && selectedCountry.states || null;
```